apiplugin/file/file_test.go

```diff
@@ -93,6 +93,14 @@ func TestCreateClose(t *testing.T) {
 	}
 }
 
+func TestOpenRead(t *testing.T) {
+	svc := newFileSvc((context.Background()))
+	creatAGoodFile(t, svc)
+	//fid := openAGoodFile(t, svc)
+
+	//testFileRead(t, svc, fid, 2, "read a file", false, int32(file.FileErr_NoError))
```

Comment:
Should this function be removed? Looks like doesn't actually test anything.

Reply:
looks like you are in an older commit, the latest one (72e1469) contains some tests